test/SlowTests/Issues/RavenDB-15900.cs

```diff
@@ -167,83 +169,102 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
         public async Task RemoveEntryFromRaftLogTest()
         {
             var (nodes, leader) = await CreateRaftCluster(3, watcherCluster: true);
-            var database = GetDatabaseName();
 
-            await CreateDatabaseInClusterInner(new DatabaseRecord(database), 3, leader.WebUrl, null);
-            using (var store = new DocumentStore
-            {
-                Database = database,
-                Urls = new[] { leader.WebUrl }
-            }.Initialize())
-            {
-                await ActionWithLeader(l => l.ServerStore.Engine.StateMachine.Validator = new TestCommandValidator());
+            await ActionWithLeader(l => l.ServerStore.Engine.StateMachine.Validator = new TestCommandValidator());
 
-                var documentDatabase = await leader.ServerStore.DatabasesLandlord.TryGetOrCreateResourceStore(database);
+            // Stuck leader on this command
+            var testCmd = new RachisConsensusTestBase.TestCommandWithRaftId("test", RaftIdGenerator.NewId());
+            _ = leader.ServerStore.Engine.CurrentLeader.PutAsync(testCmd, TimeSpan.FromSeconds(2));
 
-                var cmd = new RachisConsensusTestBase.TestCommandWithRaftId("test", RaftIdGenerator.NewId());
+            // Get last raft index from leader
+            var testCmdIndex = await WaitForCommandIndex(nodes, "TestCommandWithRaftId");
```

Comment:
why not to use the existing `LastRaftIndexForCommand` ?

Reply:
I use it inside `WaitForCommandIndex`